### ISSUE TITLE
Image alt text should be an attribute of the asset

### ISSUE THREAD

**Issue by taylortom** [adaptlearning/adapt_authoring#1118]
...so it's specified once per asset, rather than having to do so each time it's used.

**Comment by erichgoldman** [comment]
A single image may be reused in many different contexts. Therefore the alt text may be different and should not be locked based on when the asset is created. 

Perhaps it makes sense to have a default option when creating the asset or to use the existing asset title as a default, but not a disabled text field so the user can change it. This would be easier to implement as it does not require adding a new field. 

Further, it looks like in at least a few places the alt text comes before the user selects the image. Perhaps the alt text should then be put under, otherwise users will by default write the text and then have it overwritten when selecting an image. In cases where you can select different files for large and small image, this could cause some confusion as well.
